swagger-ui/src/main/java/ws/ament/hammock/swagger/ui/SwaggerUIProvider.java

```diff
@@ -0,0 +1,69 @@
+package ws.ament.hammock.swagger.ui;
```

Comment:
Missing header.  Since you're writing most of this yourself, do you want to add your name to the NOTICES file?

Reply:
Done